AppController/djinn.rb

```diff
@@ -3590,23 +3590,28 @@ def copy_encryption_keys(dest_node)
     Djinn.log_info("Copying SSH keys to node at IP address #{ip}")
     ssh_key = dest_node.ssh_key
 
+    # Commands used in the cloud environment to set the root access.
+    options = "-o StrictHostkeyChecking=no -o NumberOfPasswordPrompts=0"
+    set_authorized_keys = "sudo sort -u ~#{user_name}/.ssh/authorized_keys /root/.ssh/authorized_keys -o #{CONFIG_FILE_LOCATION}/ssh_access"
+    enable_root_login = "sudo sed -n '/.*Please login/d; w/root/.ssh/authorized_keys' #{CONFIG_FILE_LOCATION}/ssh_acces"
```

Comment:
Is `ssh_acces` a typo? Also, would it make sense to remove that file afterwards (like in tools)?